src/graph/gsubgpos-graph.hh

```diff
@@ -179,6 +179,9 @@ struct Lookup : public OT::Lookup
       if (new_sub_tables.in_error ()) return false;
       if (!new_sub_tables) continue;
       hb_pair_t<unsigned, hb_vector_t<unsigned>>* entry = all_new_subtables.push ();
+      if (all_new_subtables.in_error()) {
+        continue;
+      }
```

Comment:
An alternative approach is to use `all_new_subtables.push (hb_pair (..., ...))` instead.